components/Tabs/examples/supplemental/TabBarIconExampleSupplemental.m

```diff
@@ -22,33 +22,200 @@
 
 #import "TabBarIconExampleSupplemental.h"
 
+#import "MaterialAppBar.h"
 #import "MaterialButtons.h"
 #import "MaterialTabs.h"
+@import MaterialComponents.MaterialPalettes;
 
 @implementation TabBarIconExample (Supplemental)
 
 - (void)setupExampleViews {
```

Comment:
I took a stab at breaking it into 4. I still have one that doesn't fit within one page.